Implementation of Microsoft Graph Connector (#1)
ballerina-platform/module-microsoftgraph: src/msspreadsheets/Module.md

> Connects to a Microsoft Office 365 Workbook located on Microsoft OneDrive

Reviewer: ```suggestion
This module allows users to connect to a Microsoft Office 365 Workbook located on Microsoft OneDrive.
```

Author: Done

# Module Overview
This module contains operations to perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.

## Compatibility
|                     |    Version     |
|:-------------------:|:--------------:|
| Ballerina Language  | 1.2.0   |
| Microsoftgraph REST API | v1.0          |

## Supported Operations

> - Open workbook

Reviewer: ```suggestion
- Open a workbook
```

Author: Done

> - Create worksheet

Reviewer: ```suggestion
- Create a worksheet
```

Author: Done

> - Open worksheet

Reviewer: ```suggestion
- Open a worksheet
```

Author: Done

> - Remove worksheet

Reviewer: ```suggestion
- Remove a worksheet
```

Author: Done

> - Create table

Reviewer: ```suggestion
- Create a table
```

Author: Done

> - Rename table

Reviewer: ```suggestion
- Rename a table
```

Author: Done

> - Set Table header

Reviewer: ```suggestion
- Set a table header
```

Author: Done

> - Insert data into table

Reviewer: ```suggestion
- Insert data into a table
```

Author: Done

## Sample Application

> Please refer the sample application on the README.

Reviewer: ```suggestion
Refer the sample application on the README.
```
Shall we link this to the "README"?

Author: Based on the comment from Danesh, I will add some code snippets of supported API calls here. Hence I will remove the highlighted sentence.
